hetzner/README-cloud.md

```diff
@@ -1,17 +1,82 @@
+
+For Linux installation, binaries are available at [releases](https://github.com/hetznercloud/cli/releases). Download `hcloud-linux-amd64.tar.gz` and extract the `hcloud` 
+file into the `~/bin` directory. 
+
+All Ansible playbooks must be launched from the root of the project, at `k8s-infra`.
+
+## Init the hetzner context
+
 To getting started, you must get a Token for your API as described [here](https://docs.hetzner.cloud/#overview-getting-started).
 
 In order to create a vm and next access it, you must first import your ssh public key using this command
 ```bash
 hcloud ssh-key create --name USER_KEY_NAME --public-key-from-file ~/.ssh/id_rsa.pub
 ```
 
-## Using Ansible playbook
+Then init the hetzner context using the available `hetzner-init-context` ansible playbook.
+
+```bash
+$ ansible-playbook hetzner/ansible/hetzner-init-context.yml
+```
+
+This playbook has the following variables.
+
+| Variable | Required | Prompt | Meaning |
+| --- | :---: | :---: | --- |
+| hetzner_context_name | x | x | context name |
+| hetzner_token | x | x | The token to register with Hetzner. |
+
+Each of the Ansible prompts can be replaced by defining it's value as an extra variable of the playbook.
+
+```bash
+$ ansible-playbook hetzner/ansible/hetzner-init-context.yml -e hetzner_context_name=mycontext -e hetzner_token=mytoken 
+```
+
+The context can be verified reviewing the configuration file.
+
+```bash
+$ cat  ~/.config/hcloud/cli.toml
+
+active_context = "oneofmycontexts"
+
+[[contexts]]
+  name = "oneofmycontexts"
+  token = "zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz"
+```
+
+## Create the hetzner vm
+
+The task of creating the hetzner context is also available through an Ansible playbook.
+
+```bash
+$ ansible-playbook hetzner/ansible/hetzner-create-hcloud-server.yml
+```
+
+This will present some prompts which can be replaced by environment variables. 
+
+| Variable | Required | Default | Prompt | Meaning |
+| --- | :---: | :---: | :---: | --- |
+| vm_name | x | | x | Name of the VM to be created at hetzner. |
+| server_type | | cx31 |  | Server type. The list can be obtained using `hcloud server-type list`. Usually cx31 |
+| vm_image | | centos-7 |  | Hetzner image to be used as source (centos-7,...) | 
+| salt_text | x | | x | Salt and pepper. |
+| password_text | x | | x | Password of the created user. |
+| hetzner_context_name | x | | x | Context name for hcloud. |
+| vm_delete |  | False |  | Deletes the VM prior to creating it. |
+
+Each of the Ansible prompts can be replaced by defining it's value as an extra variable of the playbook.
+
+```bash
+$ ansible-playbook hetzner/ansible/hetzner-create-hcloud-server.yml -e '{ "vm_name": "my-name" }' -e '{ "salt_text": "<my salt>" }' -e '{ "password_text": "<my password>" }' -e '{ "hetzner_context_name": "<context name>" }' -e '{ "vm_delete": True }' -e '{ "pass_store_dir": "/home/<my home>/.my-password-store" }'
+```
+
+## Steps to create a k8s cluster
```

Comment:
I will review this section after the merge and certainly move the commands to another README file or section of a README file